src/store/flow/block/set-contact-property.ts

```diff
@@ -0,0 +1,76 @@
+import {IRootState} from '@/store'
```

Comment:
@object-required , thnx for creating this file. I'd suggest we move here all setContactProperty actions I created in `block.ts`, what do you think ?

https://github.com/FLOIP/flow-builder/blob/feature/VMO-6461/scp-array-methods/src/store/flow/block.ts#L178-L251

Reply:
Actually, I meant to suggest doing the other way around. The `block.ts` almost hit 400 lines, and splitting its contents by grouping methods logically could make it easier to maintain, just like we did with routes recently.